src/components/VerticalNavigation/__mocks__/mockIconBarChildren.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+export const mockIconBarChildren = (
+  <React.Fragment>
+    <li className="drawer-pf-trigger">
+      <a className="nav-item-iconic">
+        <span className="fa fa-bell" title="Notifications" />
+        <span className="badge badge-pf-bordered" />
+      </a>
+    </li>
+    <li className="dropdown">
```

Comment:
This should use the Dropdown component, the menu items do not show currently

Reply:
Yeah, this was just thrown in for the demo yesterday. I'll use the right components.

Reply:
It looks like this masthead is included here as just a placeholder. If so, then I'm wondering how much effort we should put into creating the mock masthead, especially if the plan is to actually build a masthead component later.